src/core/Directus/Authentication/Exception/TFAEnforcedException.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Directus\Authentication\Exception;
+
+use Directus\Exception\UnauthorizedException;
+
+class TFAEnforcedException extends UnauthorizedException
+{
+    const ERROR_CODE = 113;
+
+    public function __construct()
+    {
+        parent::__construct('2FA enforced but not activated for user');
```

Comment:
Does this need a second parameter? The error above looks like

`parent::__construct('User missing 2FA OTP', static::ERROR_CODE);`

but this one is missing `static::ERROR_CODE`

Reply:
That parameter isn't needed to extend an UnauthorizedException